pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -186,6 +186,11 @@ func (pas *PodAutoscalerStatus) MarkScaleTargetInitialized() {
 	podCondSet.Manage(pas).MarkTrue(PodAutoscalerConditionScaleTargetInitialized)
 }
 
+// IsSKSReady returns true if the PA condition denoting that SKS is ready.
+func (pas *PodAutoscalerStatus) IsSKSReady() bool {
```

Comment:
I think this isn't actually needed after latest refactor, but I guess it's a reasonable method to have anyway 🤷🏼